aws-predictions/src/main/java/com/amplifyframework/predictions/aws/models/AWSVoiceType.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.amplifyframework.predictions.models.VoiceType;
+
+/**
+ * Types of voice to synthesize speech with.
+ *
+ * @see <a href=https://docs.aws.amazon.com/polly/latest/dg/voicelist.html>Voices in Amazon Polly</a>
+ */
+@SuppressWarnings("JavadocVariable")
+public enum AWSVoiceType implements VoiceType {
+    ARABIC_ZEINA("Zeina", "arb"),
+    CHINESE_ZHIYU("Zhiyu", "cmn-CN"),
```

Comment:
```suggestion
    MANDARIN_CHINESE_ZHIYU("Zhiyu", "cmn-CN"),
```

Reply:
good suggestion! Makes sense since we are following {VARIATION}_{LANGUAGE}_{NAME} scheme